src/main/scala/firrtl/passes/CheckHighForm.scala

```diff
@@ -96,7 +96,7 @@ trait CheckHighFormLike { this: Pass =>
 
     val intModuleNames = c.modules.view.collect({ case m: Module => m.name }).toSet
 
-    c.modules.view.groupBy(_.name).filter(_._2.length > 1).flatMap(_._2).foreach {
+    c.modules.groupBy(_.name).filter(_._2.length > 1).flatMap(_._2).foreach {
```

Comment:
note of this: https://www.scala-lang.org/blog/2018/06/13/scala-213-collections.html